lib/rack/utils.rb

```diff
@@ -250,6 +251,33 @@ def add_cookie_to_header(header, key, value)
       end
     end
 
+    # :call-seq:
+    #   parse_cookies(env) -> hash
+    #
+    # Parse cookies from the provided request environment.
+    #
+    # Returns a map of cookie key to cookie value.
+    def parse_cookies(env)
+      parse_cookies_header env[HTTP_COOKIE]
+    end
+
+    # :call-seq:
+    #   set_cookie_header(key, value) -> encoded String
+    #   set_cookie_header("session", "id") -> "session=id"
+    #   set_cookie_header("session", {value: "id", max_age: 3600}) -> "session=id; max-age=3600"
```

Comment:
call-seq, if you do include it, should never include example code (such as string values).  This type of code is great, but please add it as a code block.

Reply:
Fixed.